src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -45,11 +43,11 @@ public function __construct(
 		string $merchantId,
 		string $origPayId,
 		string $orderId,
-		int $totalAmount = null,
-		Currency $currency = null,
+		Cart $cart = null,
```

Comment:
Očima lajka: Co special object, který nese jen totalAmount a currency? Pak by se dal vložit i do toho Cartu...